index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="style.css">  
+<script src="https://kit.fontawesome.com/a076d05399.js"></script>
+<title>Apple</title>
+</head>
+<body class="container">
+
+  <section class="mainsec">
+  <div class="main">
+   <nav>
+     <span><a href="#"><i class="fab fa-apple"></i></a></span>
+     <span><a href="#">Store</a></span>
+     <span><a href="#">Mac</a></span>
+     <span><a href="#">iPod</a></span>
+     <span><a href="#">iPhone</a></span>
+     <span><a href="#">iPad</a></span>
+     <span><a href="#">iTunes</a></span>
+     <span><a href="#">Support</a></span>
```

Comment:
- [OPTIONAL] While this is not wrong, I would strongly advise you to wrap everything inside the anchor tag instead of the span tag (`<a><span></a></span> instead of <span><a></a></span>`), that way all the elements can be clickable!